cluster/router/condition/router.go

```diff
@@ -117,7 +117,13 @@ func NewConditionRouter(url *common.URL) (*ConditionRouter, error) {
 	}
 
 	router.url = url
-	router.priority = url.GetParamInt(constant.RouterPriority, 0)
+	var defaultPriority int64
+	if url.GetParam(constant.APPLICATION_KEY, "") != "" {
+		defaultPriority = 150
+	} else {
+		defaultPriority = 140
```

Comment:
why change default priotiry ?

Reply:
This is the default priority for app router and service router. to ensure the service router has higher priority compare to app router if no priority is set in configuration.

Reply:
No, bro.....

You have got my point.

Look at the left block.....default priority is `0`, you change to `140`.

Your logic is if application router set priority to `150 , if not, set priority to `140`. But original logic is if not, set priority to `0`.